scripts/getConditionVars.js

```diff
@@ -0,0 +1,13 @@
+
+
+module.exports = core => {
+    const CLIENT_PAYLOAD_ARG = process.env.CLIENT_PAYLOAD_ARG;
+    try {
+        const payload = JSON.parse(CLIENT_PAYLOAD_ARG);
+        const isNonCommitEvent = payload.isNonCommitEvent || false;
+        core.exportVariable('IS_NON_COMMIT_EVENT', isNonCommitEvent);  
+    } catch (error) {
+        core.setFailed(error.message);
+        core.exportVariable('IS_NON_COMMIT_EVENT', false);  
+    }
+  };
```

Comment:
1. apply formatting
2. i think `payload.isNonCommitEvent == true` is better then `payload.isNonCommitEvent || false`, because in second use case it may come as object/string etc...

```suggestion
module.exports = core => {
  const CLIENT_PAYLOAD_ARG = process.env.CLIENT_PAYLOAD_ARG;

  try {
    const payload = JSON.parse(CLIENT_PAYLOAD_ARG);
    const isNonCommitEvent = payload.isNonCommitEvent == true;
    core.exportVariable('IS_NON_COMMIT_EVENT', isNonCommitEvent);
  } catch (error) {
    core.setFailed(error.message);
    core.exportVariable('IS_NON_COMMIT_EVENT', false);
  }
};

```

Reply:
1. wasn't able to fix formatting, it doesn't like this script
2. fixed